code/game/machinery/Sleeper.dm

```diff
@@ -190,6 +195,55 @@
 		return
 	return
 
+/obj/machinery/sleeper/Dispose()
+	occupant.in_stasis = FALSE //clean up; end stasis; remove from processing
+	occupant = null
+	processing_objects.Remove(src)
+	stop_processing()
+	return ..()
+
+/obj/machinery/sleeper/examine(mob/living/user)
+	..()
+	if(!occupant) //Allows us to reference medical files/scan reports for cryo via examination.
+		return
+	if(!ishuman(occupant))
+		return
+	var/active = ""
```

Comment:
Can you name this `active_feedback` or something that doesn't suggest it's a boolean?